vllm_ascend/attention.py

```diff
@@ -19,7 +19,7 @@
                                            PagedAttentionMetadata)
 
 if TYPE_CHECKING:
-    from vllm_ascend_plugin.worker import ModelInputForNPUBuilder
+    from vllm_ascend.worker import ModelInputForNPUBuilder
```

Comment:
This is a bug included before. I think we can fix it here as it is tiny.
```suggestion
    from vllm_ascend.model_runner import ModelInputForNPUBuilder
```